apps/i18n/common/en_us.json

```diff
@@ -523,6 +523,12 @@
   "findLocalClassButton": "Find a class",
   "finish": "Finish",
   "forTeachersOnly": "For Teachers Only",
+  "gdprDialogHeader": "Do you agree to using a web site based in the United States?",
```

Comment:
Do we need strings in apps if they're already in dashboard? I thought these had the ability to "share." But not familiar with React so not positive.

Reply:
Oooh, Brendan just told me you're thinking of making the dialog react. I think I get it now. So if we go with React it won't need to be in dashboard. Will strings still need to live in pegasus?

A) dashboard + pegasus --> apps only
B) dashboard + pegasus --> apps + pegasus

 Either way, when you're done with the dialog I'll adjust my follow up loc PR as needed.

Reply:
If I make the dialog in react, like this - then the strings only need to live in apps because the only pegasus page where we're showing the dialog is the teacher dashboard, which already has a JS entry point for React.

Reply:
nit: the strings used in sign-up have "website" rather than "web site".  Not sure which we consider correct, but we should be consistent.

Reply:
> If I make the dialog in react, like this - then the strings only need to live in apps because the only pegasus page where we're showing the dialog is the teacher dashboard, which already has a JS entry point for React.

@dju90 FYI past us had a similar question :)